playwright-e2e/pages/pancan/enrollment/consent-form-page.ts

```diff
@@ -34,30 +52,61 @@ export default class ConsentFormPage extends PancanPage {
    * <br> Type: Input
    */
   firstName(): Input {
-    return new Input(this.page, { ddpTestID: 'answer:CONSENT_FIRSTNAME' });
+    return new Input(this.page, { ddpTestID: PatientsData[this.typePatient].ddpTestID.firstName });
   }
 
   /**
    * <br> Question: Last Name
    * <br> Type: Input
    */
   lastName(): Input {
-    return new Input(this.page, { ddpTestID: 'answer:CONSENT_LASTNAME' });
+    return new Input(this.page, { ddpTestID: PatientsData[this.typePatient].ddpTestID.lastName });
   }
 
   /**
    * <br> Question: Your Signature (Full Name):
    * <br> Type: Input
    */
   signature(): Input {
-    return new Input(this.page, { ddpTestID: 'answer:CONSENT_SIGNATURE' });
+    return new Input(this.page, { ddpTestID: PatientsData[this.typePatient].ddpTestID.signature });
+  }
+
+  /**
+   * <br> Question: Your Signature (Full Name):
+   * <br> Type: Input
+   */
+  childSignature(): Input {
+    return new Input(this.page, { ddpTestID: PatientsData[this.typePatient].ddpTestID.assentChildSignature });
   }
 
   /**
    * <br> Question: AUTHORIZATION SIGNATURE
    * <br> Type: Input
    */
   authorizationSignature(): Input {
-    return new Input(this.page, { ddpTestID: 'answer:CONSENT_SELF_SIGNATURE_SUBJECT' });
+    return new Input(this.page, { ddpTestID: PatientsData[this.typePatient].ddpTestID.authorizationSignature });
+  }
+
+  async waitSecondReady(): Promise<void> {
+    await this.childSignature().toLocator().waitFor({ state: 'visible' });
+  }
+
+  /**
+   * Used in Child enrollment.
+   * @returns {Promise<void>}
+   */
+  async fillInParentData() {
+    const parentFirstName = new Input(this.page, { ddpTestID: PatientsData.child.ddpTestID.assentFirstName });
+    const parentLastName = new Input(this.page, { ddpTestID: PatientsData.child.ddpTestID.assentLastName });
+    const parentSignature = new Input(this.page, { ddpTestID: PatientsData.child.ddpTestID.assentSignature });
+    const relationshipChild = new Question(this.page, {
+      prompt: PatientsData.child.ddpTestID.relationshipChild || ''
```

Comment:
Out of curiosity, what is the ( || '' ) for? I'm reading as "relationship to child or nothing", is that close to right? I get how relationshipChild is used (as seen below and in patient-type.ts) just not entirely sure how to read the prompt.